<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D_Volume</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="11" column="2">
    <widget class="QLabel" name="label_resolution">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>resolution:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>stretch:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>min/max:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="x_lab">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>y axis</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="label_reference_data">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>reference:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="valuetext_y_min"/>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="button_flip_x">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 0px</string>
     </property>
     <property name="text">
      <string>⇄</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="label_resolution">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>resolution:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>stretch:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QLineEdit" name="valuetext_z_max"/>
   </item>
   <item row="6" column="5">
    <widget class="QLineEdit" name="valuetext_y_stretch">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="bool_downsample">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Downsample when panning</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="bool_native_aspect">
       <property name="text">
        <string>Native aspect ratio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="bool_perspective_view">
       <property name="text">
        <string>Perspective</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_line_width">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Line Width</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="value_line_width">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="bool_clip_data">
       <property name="text">
        <string>Restrict to bounding box</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="bool_visible_axes">
       <property name="text">
        <string>Show axes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>min/max:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>min/max:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QComboBox" name="combosel_x_att">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="3">
    <widget class="QComboBox" name="combosel_y_att">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QSlider" name="value_y_stretch">
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="x_lab">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>y axis</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>stretch:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QToolButton" name="button_flip_z">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 0px</string>
     </property>
     <property name="text">
      <string>⇄</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>stretch:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="valuetext_z_min"/>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="valuetext_x_stretch">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="3">
    <widget class="QComboBox" name="combosel_z_att">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="3" colspan="3">
    <widget class="QComboBox" name="combosel_reference_data"/>
   </item>
   <item row="9" column="5">
    <widget class="QLineEdit" name="valuetext_z_stretch">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QToolButton" name="button_flip_y">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 0px</string>
     </property>
     <property name="text">
      <string>⇄</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3" colspan="2">
    <widget class="QSlider" name="value_z_stretch">
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="x_lab">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>x axis</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>min/max:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3" colspan="2">
    <widget class="QComboBox" name="combosel_resolution"/>
   </item>
   <item row="5" column="5">
    <widget class="QLineEdit" name="valuetext_y_max"/>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QSlider" name="value_x_stretch">
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="x_lab">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>z axis</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="valuetext_x_max"/>
   </item>
   <item row="12" column="2" colspan="4">
    <layout class="QVBoxLayout" name="layout_slices"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="valuetext_x_min"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
